src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -0,0 +1,205 @@
+    using type = double;
+    static constexpr OptionString help = {
+        "Outer coordinate radius of Layer 1 for Object B."};
+  };
+
+  struct XCoordObjectB {
+    using type = double;
+    static constexpr OptionString help = {
+        "x-coordinate of the center for Object B."};
+  };
+
+  struct ExciseInteriorB {
+    using type = bool;
+    static constexpr OptionString help = {
+        "Exclude Layer 0 for ObjectB. Set to `true` for a BH."};
+  };
+
+  struct RadiusOuterCube {
+    using type = double;
+    static constexpr OptionString help = {
+        "Radius of Layer 3 which circumscribes the Frustums."};
+  };
+
+  struct RadiusOuterSphere {
+    using type = double;
+    static constexpr OptionString help = {"Radius of the entire domain."};
+  };
+
+  struct InitialRefinement {
+    using type = size_t;
+    static constexpr OptionString help = {
+        "Initial refinement level. Applied to each dimension."};
+  };
+
+  struct InitialGridPoints {
+    using type = size_t;
+    static constexpr OptionString help = {
+        "Initial number of grid points in each dim per element."};
+  };
+
+  struct UseEquiangularMap {
+    using type = bool;
+    static constexpr OptionString help = {
+        "Use equiangular instead of equidistant coordinates."};
+    static type default_value() { return false; }
+  };
+
+  struct UseProjectiveMap {
+    using type = bool;
+    static constexpr OptionString help = {
+        "Use projective scaling on the frustal cloak."};
+    static type default_value() { return true; }
+  };
+
+  using options =
+      tmpl::list<InnerRadiusObjectA, OuterRadiusObjectA, XCoordObjectA,
+                 ExciseInteriorA, InnerRadiusObjectB, OuterRadiusObjectB,
+                 XCoordObjectB, ExciseInteriorB, RadiusOuterCube,
+                 RadiusOuterSphere, InitialRefinement, InitialGridPoints,
+                 UseEquiangularMap, UseProjectiveMap>;
+
+  static constexpr OptionString help{
+      "The BinaryCompactObject domain is a general domain for two compact \n"
+      "objects. The user must provide the inner and outer radii of the \n"
+      "spherical shells surrounding each of the two compact objects A and B. \n"
+      "The user must also provide the radius of the sphere that \n"
+      "circumscribes the cube containing both compact objects, and the \n"
+      "radius of the outer boundary. The options ExciseInteriorA and \n"
+      "ExciseInteriorB determine whether the layer-zero blocks are present \n"
+      "inside each compact object. If set to `true`, the domain will not \n"
+      "contain layer zero for that object. The user specifies XCoordObjectA \n"
+      "and XCoordObjectB, the x-coordinates of the locations of the centers \n"
+      "of each compact object. In these coordinates, the location for the \n"
+      "axis of rotation is x=0. Please make sure that your choices of \n"
+      "locations along with the resulting center of mass is compatible with \n"
+      "this."};
```

Comment:
(non-binding) I think portions of this text are somewhat redundant with the descriptions of the individual options, and moreover, I worry someone will need to pull up a picture of the domain to know what values to pass in regardless. I suggest shortening this: I think the most valuable thing to say is (a) it's for two compact objects, (b) A is on the left, B is on the right, (c) make sure the x-coords account for the mass ratio.

Reply:
I've kept this part mostly the same in the changes because in the past I've kept the help text redundant with the options description as well. I have added a sentence about A/B left/right and to make sure that the center of mass is at x=0 though. :+1: